lib/views/invoice_slack_view.ex

```diff
@@ -4,10 +4,15 @@ defmodule Aprb.Views.InvoiceSlackView do
   def render(event) do
     partner_data = fetch_partner_data(event["properties"]["partner_id"])
     %{
-      text: ":money_with_wings: Invoice #{event["object"]["display"]} #{event["verb"]}",
+      text: ":money_with_wings: Invoice #{event["object"]["display"]}",
       attachments: "[{
                       \"fields\": [
                         {
+                          \"title\": \"Artworks\",
+                          \"value\": \"#{artworks_display_from_artworkgroups(event["properties"]["artwork_groups"])}\",
+                          \"short\": false
+                        },
+                        {
                           \"title\": \"Total\",
```

Comment:
Minor, but this manual JSON-escaping will get tiresome. Can we use a JSON lib to encode an elixir map to JSON dynamically instead?